nrcemt/qeels/gui/main_window.py

```diff
@@ -90,8 +97,32 @@ def __init__(self):
             self.canvas_click
         )
 
-        # Average Pixel
         results = ttk.Frame(settings_frame)
+
+        list_frame = ttk.Frame(results)
+        ttk.Label(
+            list_frame,
+            text="Select a material: "
+        ).pack()
+
+        dropdown_var = tk.StringVar(value=MATERIAL_OPTIONS)
+
+        self.material_list = tk.Listbox(
+            list_frame,
+            listvariable=dropdown_var,
+            height=6
+        )
+        self.material_list.select_set(0)
+        self.material_list.pack(padx=20)
+
+        list_frame.pack(side="right")
+
+        ttk.Label(
+            results,
+            text=""
+        ).pack()
```

Comment:
what is this for?

Reply:
that was a quick way to align things, ill change it now